src/components/ContributePrompt.js

```diff
@@ -82,6 +82,24 @@ export default class ContributePrompt extends Component {
     )
   }
 
+  renderEditedDefinitions = () => {
+    const { definitions } = this.props
+    return (
+      <FormGroup>
+        <ControlLabel>Involved Components</ControlLabel>
+        <ul className="definitions-list">
+          {definitions &&
+            definitions.map(definition => (
+              <li>
+                <span className="definition-name">{definition.name}</span>
```

Comment:
Can we add the namespace and the type logo too?

Reply:
I think this is a great suggestion. Will do.